### PR TITLE
densenet example

### PR DESCRIPTION
**Pull request by dfalbel**
Example discussed at #138 
Let me know what you think!

### CODE REVIEW

File: vignettes/examples/cifar10_densenet.R [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  metrics = "accuracy"
+)
+
+# Model fitting -----------------------------------------------------------
+
+# callbacks for weights and learning rate
+lr_schedule <- function(epoch) {
+  
+  if(epoch <= 150) {
+    return(0.1)
```
Comment (jjallaire) [inline vignettes/examples/cifar10_densenet.R]: Style: these don't need the explicit `return` statement (the values alone suffice)

File: vignettes/examples/cifar10_densenet.R [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  } else if(epoch > 150 & epoch <= 225){
```
Comment (jjallaire) [inline vignettes/examples/cifar10_densenet.R]: I typically use `&&` for conditional statements (although perhaps your style is better, I'm not 100% sure!)

File: vignettes/examples/cifar10_densenet.R [hunk 3]
```diff
@@ -0,0 +1,82 @@
+#' In this example we will train a DenseNet-40-12 to classify images from the
```
Comment (jjallaire) [inline vignettes/examples/cifar10_densenet.R]: Maybe link to the DenseNet paper and provide a sentence regarding motivation for DenseNet?

**Comment by dfalbel** [comment]
@jjallaire I only had time now to fix this. I think this is ready to merge. Let me know what you think.
Thanks!

**Pull request merged**